### PR TITLE
feat: switch to exodus/crypto for secp256k1

### PR DESCRIPTION
**Pull request by ChALkeR**
This a non-breaking change, `elliptic`-like `raw` format still works and passes the same tests though is doc-deprecated

This also does the following:
 * Rename `binary` to `sig|rec` for clarity (old name kept for compat):
   `Buffer.concat([signature, new Uint8Array([recovery])])`
 * Add `rec|sig` format (ethereum uses that)
   `Buffer.concat([new Uint8Array([recovery]), signature])`
 * Add `sig,rec` format (ethereum and cosmos use that)
   `{ signature, recovery }` (signature is Buffer, recovery is 0 or 1)
 * Add `sig` format (bitcoin also uses that):
   `signature` (Buffer)
 * Add `extraEntropy` option for non-schnorr secp256k1 to replace `pers`

### CODE REVIEW

File: features/keychain/module/crypto/secp256k1.js [hunk 1]
```diff
@@ -1,34 +1,45 @@
 import assert from 'minimalistic-assert'
 import * as secp256k1 from '@exodus/crypto/secp256k1'
-import elliptic from '@exodus/elliptic'
 import { mapValues } from '@exodus/basic-utils'
+import BN from 'bn.js'
```
Comment (ChALkeR) [inline features/keychain/module/crypto/secp256k1.js:4]: need this for compat with `raw` format for now

**Review by feri42 — DISMISSED** [review]
utACK - nice!

**Review by kewde — APPROVED** [review]
utACK

**Comment by ChALkeR** [comment]
Added tests for all new formats

**Comment by ChALkeR** [comment]
@feri42 @kewde reviews got dismissed when I added tests, please re-review

**Review by feri42 — APPROVED** [review]

**Review by mvayngrib — APPROVED** [review]

**Pull request merged**